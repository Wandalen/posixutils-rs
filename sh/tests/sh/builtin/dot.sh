cd tests/read_dir

. ./file2.txt
echo $?

x=value y=other_value
. ./filea.txt
echo $?