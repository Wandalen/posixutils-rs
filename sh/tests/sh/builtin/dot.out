hello
0
value
other_value
0
